deploy/ansible/roles/legion_enclave/defaults/main.yml

```diff
@@ -6,4 +6,6 @@ domain_delimiter: "-"
 namespace: "{{ enclave }}"
 source_secret_name: "{{ base_domain }}-tls"
 airflow_db_schema: "{{ enclave }}"
-airflow_db_enclave_user: "{{ enclave }}"
\ No newline at end of file
+airflow_db_enclave_user: "{{ enclave }}"
+k8s_api_retries_count: 10
```

Comment:
@kirillmakhonin please give right and clear names for configuration items.

k8s_api_retries_count -> k8s_api_retry_number_max_limit
k8s_api_retries_wait -> k8s_api_retry_delay_sec